install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -31,6 +31,14 @@ data:
     # utilization and potential performance loss in exchange for faster
     # convergence. Tweak this value according to your setup.
     rdsRefreshDelay: {{ .Values.global.refreshInterval }}
+
+    # Unix Domain Socket through which envoy communicates with NodeAgent SDS to get
+    # key/cert for mTLS. Use secret-mount files instead of SDS if set to empty. 
+    sdsUdsPath: "/tmp/sdsuds.sock"
+    
+    # How frequently should Envoy fetch key/cert from NodeAgent.
+    sdsRefreshDelay: 15s
```

Comment:
Why 15 sec ? Certs expire in hours at most. 

Isn't SDS push based ?

Reply:
it's pushed based, sdsRefreshDelay may not necessary(put here just in case for now), we could remove once e2e works.